graylog2-web-interface/src/components/event-notifications/event-notification-types/LegacyNotificationSummary.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CommonNotificationSummary from './CommonNotificationSummary';
+
+class EmailNotificationSummary extends React.Component {
+  static propTypes = {
+    type: PropTypes.string.isRequired,
+    notification: PropTypes.object,
+    definitionNotification: PropTypes.object.isRequired,
+    legacyTypes: PropTypes.object.isRequired,
+  };
+
+  static defaultProps = {
+    notification: {},
+  };
+
+  render() {
+    const { notification, legacyTypes } = this.props;
+    const configurationValues = notification.config.configuration;
+    const callbackType = notification.config.callback_type;
+    const typeConfiguration = legacyTypes[callbackType].configuration;
+
+    const summaryFields = Object.entries(typeConfiguration).map((keyAndValue) => {
```

Comment:
Please use array destructuring here:
```suggestion
    const summaryFields = Object.entries(typeConfiguration).map(([key, value]) => {
```

Reply:
:heavy_check_mark: